store/storetest/thread_store.go

```diff
@@ -498,8 +499,9 @@ func testThreadStorePermanentDeleteBatchForRetentionPolicies(t *testing.T, ss st
 	require.NoError(t, err)
 	_, _, err = ss.Thread().PermanentDeleteBatchForRetentionPolicies(nowMillis, 0, limit, model.RetentionPolicyCursor{})
 	require.NoError(t, err)
-	_, err = ss.Thread().Get(post.Id)
-	require.Error(t, err, "thread should have been deleted by team policy")
+	thread, err = ss.Thread().Get(post.Id)
+	assert.NoError(t, err)
+	assert.Nil(t, thread, "thread should have been deleted by team policy")
```

Comment:
Just checking that is correct, because the change here is the complete opposite of the previous version.

Reply:
Is it reversed? The error should be nil since the store function no longer returns an error for not found threads. The thread itself should be nil to confirm the thread doesnt' exist